components/k8s-model-server/http-proxy/requirements/production.txt

```diff
@@ -1,4 +1,6 @@
 grpcio==1.10.1
 tornado==4.5.2
 tensorflow==1.6.0
-tensorflow-serving-api==1.6.0
+tensorflow-serving-api==1.9.0
```

Comment:
Now, `tensorflow-serving-api` occurs import error: tensorflow/serving#999

Reply:
Now, it could install executable apis with the following operations:
1. pull from tensorflow/serving#999
2. bazel build //tensorflow_serving/tools/pip_package:build_pip_package
3. bazel-bin/tensorflow_serving/tools/pip_package/build_pip_package $(pwd)
4. pip install tensorflow_serving_api-undefined-py2.py3-none-any.whl

Reply:
[Resolved] Enable to use `get_model_stauts` by updating to 1.10.0-rc1.